### ISSUE TITLE
equiv does not work with bm25 ranking feature

### ISSUE THREAD

**Issue by jobergum** [vespa-engine/vespa#13205]
Looks like some wiring is missing for some of the advanced query operators, consider 
the following query using equiv
```
select * from sources * where title contains equiv("sars-cov-2","2019-nCoV");
```
Returned hit summary-features: 
```
 "title": "Pangolins Harbor <hi>SARS</hi>-<hi>CoV</hi>-<hi>2</hi>-Related Coronaviruses",
"bm25(title)": 0.0,
 "nativeRank(title)": 0.15561636953850272,
```

**Comment by arnej27959** [comment]
are those phrases inside equiv?
if so, can you test if bm25 works with a single phrase,
and whether bm25 works with equiv containing two simple terms?

**Comment by jobergum** [comment]
Phrase with bm25 works (though with a lower score than you get if you use and)

https://api.cord19.vespa.ai/search/?yql=select%20title,cord_uid,summaryfeatures%20from%20sources%20*%20where%20title%20contains%20phrase(%22sars%22,%22cov%22,%222%22)%3B&ranking=bm25t5&recall=%2Bcord_uid:0m17unfx

https://api.cord19.vespa.ai/search/?yql=select%20title,cord_uid,summaryfeatures%20from%20sources%20*%20where%20title%20contains%20%22sars%22%20and%20title%20contains%20%22cov%22%20and%20title%20contains%20%222%22%3B&ranking=bm25t5&recall=%2Bcord_uid:0m17unfx


Equiv with two terms produces 
https://api.cord19.vespa.ai/search/?yql=select%20title,cord_uid,summaryfeatures%20from%20sources%20*%20where%20title%20contains+equiv(%22sars%22,%22cov%22)%3B&ranking=bm25t5&recall=%2Bcord_uid:0m17unfx

Probably same issue with bm25 and phrase search https://github.com/vespa-engine/vespa/issues/13077 which also needs to be done for equiv?

**Comment by toregge** [comment]
This issue should have been fixed by PR #13264
